textproc/csvtk/Portfile

```diff
@@ -0,0 +1,382 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/shenwei356/csvtk 0.33.0 v
+categories          textproc
+maintainers         {macports.halostatue.ca:austin @halostatue} \
+                    openmaintainer
+license             MIT
+
+description         A cross-platform, efficient and practical CSV/TSV toolkit in Golang
+
+long_description    {*}${description}
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  ab9ed11f5512297510eb44ef86d68cffa0480511 \
+                    sha256  f325933dd2e2c4f8b83fac59df1a06b4b5b914c23c23acfa5658b676936dde9f \
+                    size    2505674
+
+build {
```

Comment:
If you're using the `golang` portgroup, you shouldn't have to explicitly specify `go build` in a `build` block like this.

Perhaps you could remove this `build` block, and instead just append arguments to the standard `go build` that the `golang` portgroup does?

Perhaps something like:

```
build.args           ./${name}
```

Reply:
I'll try this; I tried everything except ./ at the beginning of the directory to get the build working.